charts/victoria-metrics-distributed/README.md

```diff
@@ -0,0 +1,283 @@
+# Victoria Metrics Helm Chart for Setting up VMCluster on Multiple Availability Zones
+
+![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)  ![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/victoriametrics)](https://artifacthub.io/packages/helm/victoriametrics/victoria-metrics-distributed)
+[![Slack](https://img.shields.io/badge/join%20slack-%23victoriametrics-brightgreen.svg)](https://slack.victoriametrics.com/)
+
+A Helm chart for Running VMCluster on Multiple Availability Zones
+
+## Prerequisites
+
+* Install the follow packages: ``git``, ``kubectl``, ``helm``, ``helm-docs``. See this [tutorial](../../REQUIREMENTS.md).
+
+* PV support on underlying infrastructure.
+
+* Multiple availability zones.
+
+## Chart Details
+
+This chart is for setting up multiple VictoriaMetrics cluster instances on multiple [availability zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/).
+The default values suppose there are two availability zones with label `topology.kubernetes.io/zone: zone-a` and `topology.kubernetes.io/zone: zone-a`, default topology will be like below:
+![victoriametrics-distributed-topology](./victoriametrics-distributed-topology.jpg)
+
+It contains required availability zone components and global components, and other optional resources.
+
+Availability zone components:
+1. vmagent: remotes write to all availability zones to ensure data completeness.
+2. vmauth-ingest: performs `least_loaded` policy for spreading requests to vminsert instances located in the same availability zone.
+3. vmcluster: supports different setup per availability zone.
+4. vmauth-query: performs `least_loaded` policy for spreading requests to vmselect instances located in the same availability zone.
+5. vmauth-query-cross-az: performs `first_available` policy on vmselect instances of all the availability zones, prefers "local" vmselect to reduce cross zone network traffic.
```

Comment:
Can it be substituted with ` vmauth-global-query`?

Reply:
changed some namse, plz check the latest and let me know)